File: Adv.Data/Entities/WorkItem.cs
using System;
using Adv.Data.Enums;

namespace Adv.Data.Entities
{
    public class WorkItem : WorkItemBase
    {
        public WorkItemState State { get; set; }

        public Guid IterationId { get; set; } //?? Guid or String

        public Iteration Iteration { get; set; }
    }
}

File: client/src/app/sprint/_models/workItem.ts
import { WorkItemState } from "./WorkItemState";

export interface WorkItem {
    id: string;
    title: string;
    description: string;
    created: Date;
    lastModified: Date;
    priority: number;
    state: WorkItemState;
    iterationId: string;
    iteration: string;
}


